index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Bowling Challenge</title>
+</head>
+
+<body>
+  <script type="text/javascript" src="jquery-3.3.1.min.js"></script>
+  <article class="scoreboard">
+    <h2 id="player"></h2>
+    <h3>Scoreboard</h3>
+    <table id="scoreboard">
+      <tr>
+        <th>Frame</th>
+        <th>Roll</th>
+        <th>Knocked down pins</th>
+        <th>Total score</th>
+      </tr>
+    </table>
+  </article>
+
+  <article class="bowling">
+    <h3>Let's Play!</h3>
+    <div class="buttons">
+      <button id="1" type="button">1</button>
+      <button id="2" type="button">2</button>
+      <button id="3" type="button">3</button>
+      <button id="4" type="button">4</button>
+      <button id="5" type="button">5</button>
+      <button id="6" type="button">6</button>
+      <button id="7" type="button">7</button>
+      <button id="8" type="button">8</button>
+      <button id="9" type="button">9</button>
+      <button id="10" type="button">10</button>
+    </div>
+
+    <!-- <input id="player-name" type="text" placeholder="Your name...">
+      <button id="player-name-submit"type="button" name="button">Start Game!</button>
+
+      <button id="bowl-button" type="button" name="button">Bowl</button> -->
```

Comment:
This is a minor point, but try to get in the habit of removing commented out code before you submit it.  If you were to leave it in when submitting a tech test, for example, it just looks a little messy.